Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Just noticed a potential licensing issue: RPCS3 seems to be licensed under the GPL2.0 (only) licence, which is incompatible with the GPL3. Do you know if the original authors are willing to relicense this piece of code under the GPL2.0 or later licence? (If they are not, I think we will need to exclude the Skylander code from our Android builds.)

I will review the rest of the PR when the licensing situation is clarified.

Reply:
Let me look in to it

Reply:
@RipleyTom apologies for the @ notification, I just remembered you commented on the PR previously. Would you be happy with the above? I am not sure of your involvement but can see your name on the history of the Skylander code 😀

Reply:
Yes I made the usb handler of rpcs3 and the Skylander portal emulator was a proof of concept for the usb emulation layer.
I give my consent for it(IE the UI & the portal emulation internals) to be relicensed under GPL3.

Reply:
I figured it wouldn’t be as easy as this… turns out it is. Apparently this is legally allowed

Reply:
@RipleyTom To be clear, we're asking if the code can be relicensed as *GPL 2.0 or later* (i.e. GPL 2.0 and all later versions). Is that fine with you? Currently you've given permission to use the code under GPL3, but not under a hypothetical GPL4 if that is going to be released in the future.

Reply:
Yes it's fine, edited my comment.

Reply:
Thanks for that @RipleyTom, @leoetlino is this enough for you to continue your review? Will mark this conversation as resolved if so.